src/flows/components/ExportTaskOverlay/UpdateTaskBody.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {ChangeEvent, FC, useContext} from 'react'
+import {
+  Columns,
+  ComponentSize,
+  ComponentStatus,
+  EmptyState,
+  Form,
+  Grid,
+  Input,
+  InputType,
+} from '@influxdata/clockface'
+import TaskDropdown from 'src/flows/components/ExportTaskOverlay/TaskDropdown'
+import WarningPanel from 'src/flows/components/ExportTaskOverlay/WarningPanel'
+import QueryTextPreview from 'src/flows/components/ExportTaskOverlay/QueryTextPreview'
+import {OverlayContext} from 'src/flows/context/overlay'
+
+type Props = {
+  formattedQueryText: string
+}
+
+const UpdateTaskBody: FC<Props> = ({formattedQueryText}) => {
+  const {interval, handleSetEveryInterval, hasError, tasks} = useContext(
```

Comment:
instead of pulling tasks in like this from the context, as we're just passing it around in the backend, just to check it's length later, use a redux `useSelector` in here instead that returns a boolean of hasTasks